fabric/main.py

```diff
@@ -125,6 +126,7 @@ def is_classic_task(tup):
             callable(func)
             and (func not in _internals)
             and not name.startswith('_')
+            and not (inspect.isclass(func) and issubclass(func, Exception))
```

Comment:
I think what you say is totally reasonable, and would be all for excluding all classes and not just exception subclasses.  FWIW, I made this change for exceptions only because it seemed like a good combination of useful/safe.  It's useful because, for quick fabfiles, I'm betting exceptions are imported more often than classes. It's safe because even though there's probably one maniac out there that has [some crazy work flow](https://xkcd.com/1172/) that depends on the existing fabfile beahviour with respect to classes, *surely* no one is counting on the existing Exception behaviour, right?